dask_xgboost/core.py

```diff
@@ -298,6 +316,35 @@ def fit(self, X, y=None, classes=None):
             Feature Matrix. May be a dask.array or dask.dataframe
         y : array-like
             Labels
+        eval_set : list, optional
+            A list of (X, y) tuple pairs to use as validation sets, for which
+            metrics will be computed.
+            Validation metrics will help us track the performance of the model.
+        sample_weight_eval_set : list, optional
+            A list of the form [L_1, L_2, ..., L_n], where each L_i is a list of
+            instance weights on the i-th validation set.
+        eval_metric : str, list of str, or callable, optional
+            If a str, should be a built-in evaluation metric to use. See
+            doc/parameter.rst.
+            If a list of str, should be the list of multiple built-in evaluation metrics
+            to use.
+            If callable, a custom evaluation metric. The call
+            signature is ``func(y_predicted, y_true)`` where ``y_true`` will be a
+            DMatrix object such that you may need to call the ``get_label``
+            method. It must return a str, value pair where the str is a name
+            for the evaluation and value is the value of the evaluation
+            function. The callable custom objective is always minimized.
+        early_stopping_rounds : int
+            Activates early stopping. Validation metric needs to improve at least once in
+            every **early_stopping_rounds** round(s) to continue training.
+            Requires at least one item in **eval_set**.
+            The method returns the model from the last iteration (not the best one).
+            If there's more than one item in **eval_set**, the last entry will be used
+            for early stopping.
+            If there's more than one metric in **eval_metric**, the last metric will be
+            used for early stopping.
+            If early stopping occurs, the model will have three additional fields:
+            ``clf.best_score``, ``clf.best_iteration`` and ``clf.best_ntree_limit``.
```

Comment:
Do these have trailing underscores in xgboost?

Reply:
It doesn't look like it from [here](https://github.com/dmlc/xgboost/blob/master/python-package/xgboost/sklearn.py#L1181).